packages/core/src/components/card/card.tsx

```diff
@@ -3,7 +3,7 @@ import { h, Component, ComponentInterface, Element, Event, EventEmitter, Host, P
 import { isInteractiveElement } from "../../utils/is-interactive-element";
 import { isModifiedEvent } from "../../utils/is-modified-event";
 
-import { DsoCardClickedEvent, ImageShape } from "./card.interfaces";
+import { DsoCardClickedEvent } from "./card.interfaces";
```

Comment:
Zullen we in dit PR ook maar meteen de naam van deze interface hernoemen? Hoe de verleden tijd in deze naam terecht is gekomen...

Reply:
Ok. Niet vergeten ook in de blogpost te benoemen